### ISSUE TITLE
RESPONSE 400 - azure storage

### ISSUE THREAD

**Issue by AdrianOsica** [orgrim/pg_back#126]
When try list or download backup from azure return `RESPONSE 400`

```
root@radius:/etc/postgresql/14/main# pg_back -c pg_back.conf  --list-remote azure
2024/07/29 12:51:13 FATAL: could not list contents of remote location: could not fully list Azure container radius: GET https://sqlbackup.blob.core.windows.net/radius/radius
--------------------------------------------------------------------------------
RESPONSE 400: 400 The requested URI does not represent any resource on the server.
ERROR CODE: InvalidUri
--------------------------------------------------------------------------------
﻿<?xml version="1.0" encoding="utf-8"?>
<Error><Code>InvalidUri</Code><Message>The requested URI does not represent any resource on the server.
RequestId:7c71fbb0-001e-001e-0ca5-e1e20e000000
Time:2024-07-29T10:51:13.6431138Z</Message></Error>
--------------------------------------------------------------------------------
```

Uploud seem works

**Comment by AdrianOsica** [comment]
```
backup_directory = /var/backups/postgresql/main
port = 5432
checksum_algorithm = sha256
encrypt = true
cipher_pass = XXXXXX
purge_older_than = 90
upload = azure

azure_container = radius
azure_account = XXXXX
azure_key = XXXX

```

**Comment by dpunkturban** [comment]
Hi @AdrianOsica, I can reproduce it. I will try to fix it :)

**Comment by AdrianOsica** [comment]
i think so, microsoft azure storage explorer also use this url

![image](https://github.com/user-attachments/assets/8614b85f-18b2-4ee7-8269-d876769aa14e)

**Comment by dpunkturban** [comment]
Hi Adrian,

I've fixed the issue in #128. You can test it locally by cloning my fork and checking out the branch, or you can wait until the review is complete and the new Docker image is published.
